teams/twitter.toml

```
@@ -0,0 +1,12 @@
+name = "twitter"
+kind = "marker-team"
+
+[people]
+leads = []
+members = [
+    "m-ou-se",
+    "joshtriplett",
+    "rylev",
+    "badboy",
+    "pietroalbini"
+]
```

```suggestion
    "pietroalbini",
]
```

Trailing comma, and ensuring a final newline.